### PR TITLE
feat(Gmail): add Lang Strings

### PR DESCRIPTION
**Pull request by EncryptedDev**

### CODE REVIEW

File: src/Presence/gmail.json [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+      "message": "Viewing the Promotions category",
+      "description": "Displayed when viewing promotions emails."
+   },
+   "gmail.categoryForum": {
+      "message": "Viewing the Forum category",
+      "description": "Displayed when viewing forum emails."
+   },
+   "gmail.inLabel": {
+      "message": "Viewing label",
+      "description": "Displayed when a user is browsing emails in certain label"
```
Comment (ImRodry) [inline src/Presence/gmail.json:24]: Assuming this is true
```suggestion
      "description": "Displayed when a user is browsing emails in a certain label. The name of the label will appear below."
```

File: src/Presence/gmail.json [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+      "message": "Configuring Inbox settings",
+      "description": "Displayed when the user is modifying their inbox settings"
+   },
+   "gmail.accountSetting": {
+      "message": "Configuring Account and Import settings",
+      "description": "Displayed when the user is modifying their account's settings"
+   },
+   "gmail.filterSettings": {
+      "message": "Configuring Filters settings",
+      "description": "Displayed when the user is modifying their filters"
```
Comment (ImRodry) [inline src/Presence/gmail.json:44]:
```suggestion
      "message": "Configuring Filters and Blocked Addresses settings",
      "description": "Displayed when the user is modifying their filters and blocked addresses"
```

File: src/Presence/gmail.json [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+   },
+   "gmail.fwdandpopSettings": {
```
Comment (ImRodry) [inline src/Presence/gmail.json:46]: camel casing for you
```suggestion
   "gmail.fwdAndPOPSettings": {
```

File: src/Presence/gmail.json [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+      "message": "Configuring Forwarding and POP/IMAP settings",
+      "description": "Displayed when the user is modifying their IMAP and POP settings"
```
Comment (ImRodry) [inline src/Presence/gmail.json:48]: :)
```suggestion
      "description": "Displayed when the user is modifying their forwarding, POP and IMAP settings"
```

File: src/Presence/gmail.json [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+   },
+   "gmail.chatSettings": {
+      "message": "Configuring Chat and Meet settings",
+      "description": "Displayed when the user is modifying their chat settings"
+   },
+   "gmail.addonsSettings": {
+      "message": "Configuring Add-ons",
+      "description": "Displayed when the user is modifying their add-ons"
+   },
```
Comment (ImRodry) [inline src/Presence/gmail.json:57]: To match the order in gmail at least
```suggestion
   "gmail.addonsSettings": {
      "message": "Configuring Add-ons",
      "description": "Displayed when the user is modifying their add-ons"
   },
   "gmail.chatSettings": {
      "message": "Configuring Chat and Meet settings",
      "description": "Displayed when the user is modifying their chat settings"
   },
```

File: src/Presence/gmail.json [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+   "gmail.advancedSettings": {
+      "message": "Configuring Advanced settings",
+      "description": "Displayed when the user is modifying their advanced settings"
+   },
+   "gmail.offlineSettings": {
+      "message": "Configuring Offline settings",
+      "description": "Displayed when the user is modifying their offline settings"
+   },
+   "gmail.oldThemesSettings": {
```
Comment (ImRodry) [inline src/Presence/gmail.json:66]: why is it old

File: src/Presence/gmail.json [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+      "message": "Configuring Themes",
+      "description": "Displayed when the user is modifying their themes settings"
```
Comment (ImRodry) [inline src/Presence/gmail.json:68]:
```suggestion
      "message": "Picking a theme",
      "description": "Displayed when the user is modifying their theme in the settings"
```

File: src/Presence/gmail.json [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+   },
+   "gmail.lookingForEmail": {
+      "message": "Looking for an email",
+      "description": "Displayed when the user is looking for mail"
```
Comment (ImRodry) [inline src/Presence/gmail.json:72]: Also, will this display the contents of the search box?
```suggestion
      "description": "Displayed when the user is searching for an email"
```

File: src/Presence/gmail.json [hunk 9]
```diff
@@ -0,0 +1,126 @@
...
+   },
+   "gmail.viewingAnEmail": {
+      "message": "Viewing an email",
+      "description": "Displayed when the user is viewing mail"
```
Comment (ImRodry) [inline src/Presence/gmail.json:76]: Does this display any information about the email? (title, sender, etc)
```suggestion
      "description": "Displayed when the user is viewing an email"
```

File: src/Presence/gmail.json [hunk 10]
```diff
@@ -0,0 +1,126 @@
...
+   },
+   "gmail.oldThemesSettings": {
+      "message": "Configuring Themes",
+      "description": "Displayed when the user is modifying their themes settings"
+   },
+   "gmail.lookingForEmail": {
+      "message": "Looking for an email",
+      "description": "Displayed when the user is looking for mail"
+   },
+   "gmail.viewingAnEmail": {
```
Comment (ImRodry) [inline src/Presence/gmail.json:74]: you know you can simplify these keys
```suggestion
   "gmail.viewingEmail": {
```

File: src/Presence/gmail.json [hunk 11]
```diff
@@ -0,0 +1,126 @@
...
+      "message": "Viewing an email",
+      "description": "Displayed when the user is viewing mail"
+   },
+   "gmail.viewingInbox": {
+      "message": "Viewing inbox",
+      "description": "Displayed when the user is viewing their inbox"
+   },
```
Comment (ImRodry) [inline src/Presence/gmail.json:81]: Viewing the inbox is the same as viewing the primary category
```suggestion
```

File: src/Presence/gmail.json [hunk 12]
```diff
@@ -0,0 +1,126 @@
...
+   "gmail.viewingStarredEmails": {
+      "message": "Viewing starred emails",
+      "description": "Displayed when the user is viewing their starred emails"
```
Comment (ImRodry) [inline src/Presence/gmail.json:84]:
```suggestion
      "description": "Displayed when the user is viewing emails marked with a star"
```

File: src/Presence/gmail.json [hunk 13]
```diff
@@ -0,0 +1,126 @@
...
+   },
+   "gmail.viewingSentEmails": {
+      "message": "Viewing sent emails",
+      "description": "Displayed when the user is viewing their sent emails"
```
Comment (ImRodry) [inline src/Presence/gmail.json:88]:
```suggestion
      "description": "Displayed when the user is viewing the emails they have sent"
```

File: src/Presence/gmail.json [hunk 14]
```diff
@@ -0,0 +1,126 @@
...
+      "message": "Viewing snoozed emails",
+      "description": "Displayed when the user is viewing their snoozed emails"
+   },
+   "gmail.viewingDrafts": {
+      "message": "Viewing drafts",
+      "description": "Displayed when the user is viewing their drafts"
+   },
+   "gmail.viewingImportantEmails": {
+      "message": "Viewing important emails",
+      "description": "Displayed when the user is viewing their important emails"
```
Comment (ImRodry) [inline src/Presence/gmail.json:100]:
```suggestion
      "description": "Displayed when the user is viewing emails marked as important"
```

**Review by ImRodry — CHANGES_REQUESTED** [review]

File: src/Presence/gmail.json (hunk 6 above)
Comment (EncryptedDev) [reply src/Presence/gmail.json:66]: Just is

**Comment by ImRodry** [comment]
I can respect your commitment to being a bitch

File: src/Presence/gmail.json (hunk 9 above)
Comment (EncryptedDev) [reply src/Presence/gmail.json]: no my highness

File: src/Presence/gmail.json (hunk 8 above)
Comment (EncryptedDev) [reply src/Presence/gmail.json]: no  my highness

Comment (ImRodry) [reply src/Presence/gmail.json]: thank you servant

File: src/Presence/gmail.json (hunk 9 above)
Comment (ImRodry) [reply src/Presence/gmail.json]: thank you servant

**Pull request closed without merging**